src/test/miner_tests.cpp

```diff
@@ -203,6 +197,11 @@ static void TestPackageSelection(const CChainParams& chainparams, const CScript&
 // NOTE: These tests rely on CreateNewBlock doing its own self-validation!
 BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
 {
+    const auto TestSequenceLocks = [this](const CTransaction& tx, int flags) {
```

Comment:
In commit "test: Replace recursive lock with AssertLockHeld" (fa19ddc2bf8ee8f1ce621611a90f8b9e380b1517)

Instead of `TestSequenceLocks` being a lambda and `TestPackageSelection` in fafe3d88d34f04c798d000ca0b315830d63a80eb taking an unconventionally named `m_node` argument, I think it might be better if both were test fixture methods:

```c++
class MinerTest : public TestingSetup
{
    bool TestSequenceLocks(...);
    void TestPackageSelection(...);
};

bool MinerTest::TestSequenceLocks(...) {...}
void MinerTest::TestPackageSelection(...) {...}

BOOST_FIXTURE_TEST_CASE(CreateNewBlock_validity, MinerTest) { ... }
```

This should make the diffs here smaller and scripted diff simpler because `TestSequenceLocks` wouldn't need to move and `TestPackageSelection` arguments wouldn't have to change.

Reply:
Fixed, I think